src/utils/member/utils.js

```diff
@@ -48,3 +48,385 @@ export const updateImageUrlForMemberList = ({ members = [], dpImageUrlList = []
  */
 export const getMemberGridColumnWidth = ({ widthInteger, widthAdjustPercent }) =>
   `${(((100 - Number(widthAdjustPercent)) * Number(widthInteger)) / 100)}%`;
+
+/**
+ * This util function mark an object as empty if it's all field is undefined
+ * or null. You also skip values that you don't want to check, just simply
+ * pass that value key into the exclude field array.
+ * @param {Object} data
+ * @param {Array} excludeField
+ * @return {boolean}
+ */
+export const isObjectEmpty = (data = {}, excludeField = []) => {

```

Comment:
We can rename it to something like
```suggestion
export const isObjectRequiredKeysUnavailable = (data = {}, excludeField = []) => {

```

Reply:
done